.env-dev.template

```diff
@@ -38,6 +38,16 @@ PUBLIC_URL_PREFIX= # Set Tunnel URL if using local storage
 # AZURE_STORAGE_ACCOUNT_KEY=
 # AZURE_STORAGE_CONTAINER=
 
+# Set AWS storage keys if using AWS S3 Storage
+# AWS_S3_BUCKET_NAME=
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
+# AWS_REGION=
+
+# Set GCP credentials if using GCP Storage
+# GCP_BUCKET_NAME=
```

Comment:
Add STORAGE_TYPE=gcp for gcp storage and add STORAGE_TYPE=azure for azure storage in both .env-dev and .env-dev.template files.